services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from db.models import CinemaHall
+from typing import List
+
+
+def get_cinema_halls() -> List[CinemaHall]:
+    return CinemaHall.objects.all()
```

Comment:
Return type hint here will be `QuerySet` 

Remember:`.all()`, `.filter()` returns QuerySet`
.get()`, `.delete()`, `.create()` returns model class, `-> User` for. example